plugin/src/main/scala/AkkaVizPlugin.scala

```diff
@@ -14,15 +13,23 @@ object AkkaVizPlugin extends AutoPlugin {
 
   override def projectConfigurations: Seq[Configuration] = Seq(SbtAspectj.Aspectj)
 
-  object autoImport {
-
+  object Keys {
+    val akkaVizVersion = settingKey[String]("Version of akka-viz to use")
   }
 
-  override def projectSettings: Seq[Def.Setting[_]] = SbtAspectj.aspectjSettings ++ Seq(
-    resolvers += Resolver.bintrayRepo("lustefaniak", "maven"),
-    libraryDependencies += "com.blstream.akkaviz" %% "monitoring" % BuildInfo.version % "runtime",
-    fork := true,
-    javaOptions <++= AspectjKeys.weaverOptions in Aspectj
-  )
-
-}
+  val autoImport = Keys
+
+  override def projectSettings: Seq[Def.Setting[_]] = {
+    import Keys._
+    SbtAspectj.aspectjSettings ++ Seq(
+      resolvers += Resolver.bintrayRepo("lustefaniak", "maven"),
+      akkaVizVersion := "0.1.5",
```

Comment:
My plan is to decouple sbt plugin version from the version of library. So you won't have to upgrade plugin every time new version of library is released.